luaui/Widgets/gui_advplayerslist.lua

```diff
@@ -1119,11 +1119,11 @@ local function GetSkill(playerID)
             if osSigma then
                 osSigma = tonumber(osSigma)
                 -- 0.00 is absolute certain , 8.33 is initial uncertaintiy at registration time (written at 2024/01/11)
-                if osSigma > 6.24 then
+                if osSigma >= 5.0 then
```

Comment:
These magic values could become named constants. Perhaps a tiny config file since it's used in both a gadget and a widget already?

Reply:
Sounds good - where would i place this config file ?

Reply:
idk, luarules/configs?

Reply:
asking, because it would be used by luaui and luarules as well. 
I'll use luarules/configs then...